pages/2017/index.rst

```diff
@@ -8,4 +8,53 @@ Python in Astronomy 2017
 Lorentz Center, Leiden
 ----------------------
 
-WOOOO YEAG
+The third edition of the Python in Astronomy Conference will be held at the
+Lorentz Center on the **8 - 12th May 2017**.
+
+As with the two previous Python in Astronomy conferences we aim to bring a wide
+variety of people who use, develop or teach people about Python packages in the
+context of all forms of Astronomy. The conference will include presentations,
+tutorials unconference sessions and sprints. As well as building the community
+around astronomical uses of Python the conference aims to improve collaboration
+and interoperability between Python packages, share knowledge on Python packages
+and techniques and provide training and educational materials for users and
+developers of Python packages.
+
+Attending PyAstro17
+###################
+
+Applications for Python in Astronomy 2017 will open on the September 12th 2016,
+participants will be selected both on merit and to ensure a diversity at the
+conference.
+
```

Comment:
I think we should avoid using the word “application”. Here’s my quick and
dirty first try at some language. It is particularly aimed at the folks who
are not sure if they are welcome or not…the existing developers will
request to attend regardless of what this text says.

On Sept 12, 2016, we will begin accepting requests from interested
participants. Participant selection will be made with the goal of *growing*
the Python in Astronomy community and we particularly encourage requests to
attend from junior astronomers and people who are new to Python. Effort
will also be made to select participants who have a demonstrated potential
to contribute meaningfully to the Python in Astro infrastructure via
providing educational materials, documentation, and/or coding.

On Wed, Jul 27, 2016 at 10:15 AM, Daniela Huppenkothen <
notifications@github.com> wrote:

> In pages/2017/index.rst
> <https://github.com/OpenAstronomy/pyastro/pull/1#discussion_r72446830>:
>
> > +As with the two previous Python in Astronomy conferences we aim to bring a wide
> > +variety of people who use, develop or teach people about Python packages in the
> > +context of all forms of Astronomy. The conference will include presentations,
> > +tutorials unconference sessions and sprints. As well as building the community
> > +around astronomical uses of Python the conference aims to improve collaboration
> > +and interoperability between Python packages, share knowledge on Python packages
> > +and techniques and provide training and educational materials for users and
> > +developers of Python packages.
> > +
> > +Attending PyAstro17
> > +###################
> > +
> > +Applications for Python in Astronomy 2017 will open on the September 12th 2016,
> > +participants will be selected both on merit and to ensure a diversity at the
> > +conference.
> > +
>
> Maybe add when applications close, too? Otherwise people might miss that
> deadline ...
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenAstronomy/pyastro/pull/1/files/b7e164dc00d89f8f56fef20e4ca6fa5abc35d7fd#r72446830>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABYJmWlJE6sF7WhzGpzDlLvD0m6wgFKuks5qZ2gOgaJpZM4JU92Q>
> .
>



-- 
Kelle Cruz, PhD — http://kellecruz.com/
917.725.1334 — Hunter: x16486 — AMNH: x3404